src/common/techstack/TechStackInfo.js

```diff
@@ -15,6 +15,7 @@ import {
   SiNodedotjs,
 } from "react-icons/si";
 import NHost from "images/nhost.svg";
+import Umami from "images/umami.svg";
```

Comment:
The image needs to be little bigger in order to be in sync with others.  Refer https://analytics.rip/images/tools/umami.svg

Also can we give it a grayish look? On hover show the actual color

Reply:
Done, I have Changed the Image.

Reply:
yeah that'sd correct however my point was to make initial one with less opacity and hover will make the `opacity: 1`